src/components/Search.js

```diff
@@ -0,0 +1,10 @@
+export const SearchBar = ({ handleChange, inputValue }) => {
+  return (
+    <input
+      onChange={handleChange}
+      className="rounded-lg border mx-3.5 p-1 w-64"
+      placeholder="Username"
```

Comment:
this should also be dynamic, should be received from props.

Reply:
Done